packages/functional-tests/tests/misc/keyStretchingV2.spec.ts

```diff
@@ -182,7 +179,9 @@ test.describe('key-stretching-v2', () => {
     async function _resetPassword(
       version: 1 | 2,
       key: string | undefined,
-      opts: Pick<Opts, 'page' | 'target' | 'settings' | 'resetPasswordReact'>
+      opts: Pick<Opts, 'page' | 'target' | 'settings' | 'resetPasswordReact'>,
+      email: string,
```

Comment:
Adding these as params is fine. But using scoped variables also seems fine. Is there a reason this change is needed?

Reply:
Answered [here](https://github.com/mozilla/fxa/pull/16726#discussion_r1568784283)